python/MooseDocs/extensions/media.py

```diff
@@ -99,6 +99,7 @@ def defaultSettings():
         settings = command.CommandComponent.defaultSettings()
         settings['latex_src'] = (None, "Image to utilize when rendering with LaTeX")
         settings['dark_src'] = (None, "Image to utilize with dark HTML theme")
+        settings['link'] = (None, "Anchor to navigate to upon being clicked")
```

Comment:
```suggestion
        settings['link'] = (None, "Anchor URL to navigate to upon being clicked")
```

Does this make more sense? At a glance, one might not know what "anchor" means.